ci/kokoro/macos/builds/cmake-vcpkg.sh

```diff
@@ -37,9 +37,14 @@ io::log_h2 "Using CMake version"
 cmake --version
 
 # Fetch vcpkg at the specified hash.
-vcpkg_dir="${HOME}/vcpkg-quickstart"
+if  [[ -n "${KOKORO_ARTIFACTS_DIR:-}" ]]; then
+  vcpkg_dir="${HOME}/vcpkg-quickstart"
+else
+  vcpkg_dir="${KOKORO_ARTIFACTS_DIR}/vcpkg-quickstart"
+fi
```

Comment:
For what it's worth, this looks backwards.

Reply:
Right you are